pkg/expr/expr.md

```diff
@@ -0,0 +1,49 @@
+# Expression Language   (CEXL)
+
+Istio (mixer) configuration uses an expression language to evaluate selectors and input maps.
+The syntax of the expression language is C-like.
+
+An expression maps typed Attribute Values to a Value. 
+
+```
+f(attrValue0, attrValue1, ..., constant0, constat1, ...) --> Value
+```
+
+## Predicates
```

Comment:
call this `Selector Expressions` to match the `Mapping Expressions` header and example below.